style.css

```diff
@@ -368,33 +372,195 @@ ul {
 
 @media (max-width: 950px) {
   .grid-container {
+    marin-left: 500px;
     grid-template-columns: 1fr 1fr; /* Two columns for small screens */
     grid-template-rows: repeat(
-      8,
+      3,
       100px
     ); /* Set fixed height for small screens */
   }
 }
 
 .carousel-control-prev,
 .carousel-control-next {
-    height: 150px; 
-    width: 150px;  
+  height: 150px;
+  width: 20px;
 }
 
+.center-quote {
+  display: flex;
+  justify-content: center;
+  margin-top: 20px; /* Add margin to move the quotes down */
+}
 
-.flag-container {
+.carousel-control {
+  position: absolute;
+  //top: 20%;
+  transform: translateY(-50%);
+  font-size: 24px;
+  background: white;
+  color: black;
+  border: none;
+  cursor: pointer;
+  background: #fffff;
+}
+
+.carousel-control-prev {
+  left: 10px;
+}
+
+.carousel-control-next {
+  right: 10px;
+}
+
+.carousel-inner {
+  //display: flex;
+  transition: transform 0.5s ease-in-out;
+  overflow: hidden;
+  position: relative;
+}
+
+.carousel-item {
+  max-width: 100%;
+  min-width: 100%;
+  box-sizing: border-box;
+}
+.carousel-arrow {
+  width: 30px;
+  height: 30px;
+}
+#langs {
+  //display: flex;
+  justify-content: space-between;
+  //align-items: center;
+  padding: 20px;
+  background-color: #f2f2f2;
+}
+.gallery-text {
+  font-size: 18px;
+  //margin-bottom: 20px;
+}
+.grid-item img {
+  width: 80px;
+  height: 60px;
+  margin-bottom: 5px;
+  border-radius: 5px;
+}
+
+.flag-label {
+  margin-top: 20px;
+  margin-left: 5px;
+  display: block;
+}
+.header-text {
+  //max-width: 300px;
+  // margin-bottom: 100px;
+  justify-content: center;
+  text-allign: center;
+  margin: auto;
+}
+.blockquote {
+  font-size: 24px;
+  line-height: 36px;
+  position: relative;
+}
+
+.cite {
+  position: absolute;
+  top: 20px;
+  left: 60%;
+  transform: translateX(-50%);
+  font-size: 16px;
+}
+
+#contact-btn-container {
   display: flex;
   justify-content: center;
-  
-  height: 100vh;
+  align-items: center;
+  height: 20vh;
+  background: #fffff;
+}
+
+#contact-btn {
+  background-color: #ff6b6b;
+  color: #fdfffc;
+  padding: 15px 30px;
+  font-size: 18px;
+  border: none;
+  cursor: pointer;
+}
+
+/* Modal Styles */
+#contact-btn {
+  background-color: #ff6b6b;
+  color: #fdfffc;
+  padding: 15px 30px;
+  font-size: 18px;
+  border: none;
+  cursor: pointer;
+}
+
+/* Modal Styles */
+#contact-modal {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  justify-content: center;
+  align-items: center;
+}
+
+#modal-content {
+  background-color: #fdfffc;
+  padding: 20px;
+  border-radius: 10px;
+  text-align: center;
 }
 
-.center-quote{
+#close-modal {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  cursor: pointer;
+  font-size: 24px;
+  padding: 10px;
+  background: white;
+  border: none;
+  color: #2d232e;
+}
+
+#contact-btn {
```

Comment:
this id selector has been used twice with the same styling - delete this instance?